scripts/ci/ci_count_changed_files.sh

```diff
@@ -42,7 +42,7 @@ echo
 echo
 echo "Changed files matching the ${2} pattern"
 echo
-echo "${CHANGED_FILES}" | grep "${2}" || true
+echo "${CHANGED_FILES}" | grep -E "${2}" || true
 echo
 
 echo
```

Comment:
You also need to add -E to the grep below which actually counts the files.

Reply:
> FYI. @ashb It's enough to set VERBOSE_COMMANDS="true" in the workflow and all commands will be printed.

Oh yes, I thought this was deeper in the call stack, so I thought that would have given me a lot more logs than I want. (Plus sometimes the more direct option is easier to remember)